packages/twenty-front/src/modules/ui/navigation/navigation-drawer/components/MultiWorkspaceDropdownButton.tsx

```diff
@@ -0,0 +1,123 @@
+  height: 16px;
+  width: 16px;
+`;
+
+const StyledContainer = styled.div`
+  align-items: center;
+  cursor: pointer;
+  color: ${({ theme }) => theme.font.color.primary};
+  border-radius: ${({ theme }) => theme.border.radius.sm};
+  border: 1px solid transparent;
+  display: flex;
+  justify-content: space-between;
+  height: ${({ theme }) => theme.spacing(7)};
+  padding: 0 ${({ theme }) => theme.spacing(2)};
+  width: 100%;
+
+  &:hover {
+    background-color: ${({ theme }) => theme.background.transparent.lighter};
+    border: 1px solid ${({ theme }) => theme.border.color.medium};
+  }
+`;
+
+const StyledLabel = styled.div`
+  align-items: center;
+  display: flex;
+  gap: ${({ theme }) => theme.spacing(2)};
+`;
+
+const StyledIconChevronDown = styled(IconChevronDown)<{ disabled?: boolean }>`
+  color: ${({ disabled, theme }) =>
+    disabled ? theme.font.color.extraLight : theme.font.color.tertiary};
+`;
+
+type MultiWorkspaceDropdownButtonProps = {
+  workspaces: Workspaces[];
+};
+
+export const MultiWorkspaceDropdownButton = ({
+  workspaces,
+}: MultiWorkspaceDropdownButtonProps) => {
+  const theme = useTheme();
+  const currentWorkspace = useRecoilValue(currentWorkspaceState());
+
+  const [isMultiWorkspaceDropdownOpen, setToggleMultiWorkspaceDropdown] =
+    useState(false);
+
+  const { switchWorkspace } = useWorkspaceSwitching();
+
+  const { closeDropdown } = useDropdown(MULTI_WORKSPACE_DROPDOWN_ID);
+
+  const handleChange = async (workspaceId: string) => {
+    setToggleMultiWorkspaceDropdown(!isMultiWorkspaceDropdownOpen);
+    closeDropdown();
+    await switchWorkspace(workspaceId);
+  };
+
+  return (
+    <Dropdown
+      dropdownId={MULTI_WORKSPACE_DROPDOWN_ID}
+      dropdownHotkeyScope={{
+        scope: NavigationDrawerHotKeyScope.MultiWorkspaceDropdownButton,
+      }}
+      clickableComponent={
+        <StyledContainer>
+          <StyledLogo
+            logo={
+              currentWorkspace?.logo === null
+                ? 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAACXBIWXMAAA7EAAAOxAGVKw4bAAACb0lEQVR4nO2VO4taQRTHr3AblbjxEVlwCwVhg7BoqqCIjy/gAyyFWNlYBOxsfH0KuxgQGwXRUkGuL2S7i1barGAgiwbdW93SnGOc4BonPiKahf3DwXFmuP/fPM4ZlvmlTxAhCBdzHnEQWYiv7Mr4C3NeuVYhQYDPzOUUQgDLBQGcLHNhvQK8DACPx8PTxiqVyvISG43GbyaT6Qfpn06n0m63e/tPAPF4vJ1MJu8kEsnWTCkWi1yr1RKGw+GDRqPBOTfr44vFQvD7/Q/lcpmaaVQAr9fLp1IpO22c47hGOBz+MB6PH+Vy+VYDAL8qlUoGtVotzOfzq4MAgsHgE/6KojiQyWR/bKVSqbSszHFM8Pl8z1YK48JsNltCOBwOnrYLO+8AAIjb+nHbycoTiUQfDJ7tFq4YAHiVSmXBxcD41u8flQU8z7fhzO0r83atVns3Go3u9Xr9x0O/RQXo9/tsIBBg6vX606a52Wz+bZ7P5/WwG29gxSJzhKgA6XTaDoFNF+krFAocmC//4yWEcSf2wTm7mCO19xFgSsKOLI16vV7b7XY7mRNoLwA0JymJ5uQIzgIAuX5PzDElT2m+E8BqtQ4ymcx7Yq7T6a6ZE4sKgOadTucaCwkxp1UzlEKh0GDxIXOwDWHAdi6Xe3swQDQa/Q7mywoolUpvsaptymazDWKxmBHTlWXZm405BFZoNpuGgwEmk4mE2SGtVivii4f1AO7J3ZopkQCQj7Ar1FeRChCJRJzVapX6DKNIfSc1Ax+wtQWQ55h6bH8FWDfYV4fO3wlwDr0C/BcADYiTPCxHqIEA2QsCZAkAKnRGkMbKN/sTX5YHPQ1e7SkAAAAASUVORK5CYII='
```

Comment:
you used to have this as a variable, I feel it was more elegant :)

Reply:
Declared it as a constant now that we want to use it in many places